src/Components/CreateImageWizard/utilities/generateDefaultName.ts

```diff
@@ -1,15 +1,12 @@
-import { ImageTypes } from '../../../store/imageBuilderApi';
-
-export const generateDefaultName = (
-  distribution: string,
-  arch: string,
-  targetEnvironments: ImageTypes[]
-) => {
+export const generateDefaultName = (distribution: string, arch: string) => {
   const date = new Date();
   const day = date.getDate().toString().padStart(2, '0');
-  const month = date.toLocaleString('en-US', { month: 'long' });
-  const year = date.getFullYear();
-  return `${distribution} ${arch} ${targetEnvironments.join(
-    ' '
-  )} ${day} ${month} ${year}`;
+  const month = (date.getMonth() + 1).toString().padStart(2, '0');
+  const year = date.getFullYear().toString().slice(-2);
```

Comment:
Ah yeah, sorry, jumping all over the place this morning so getting to this a bit later. This would be the only change and I believe it would improve readability of the date, making it a bit more obvious.

```suggestion
  const year = date.getFullYear().toString();
```

Otherwise looks and works great!

Reply:
I think it is good idea to add the year.
I added it like that `${month}/${day}/${year}-${hours}:${minutes}` to be more clear

Reply:
I'd not do that, that will confuse every European out there, that's why it's not common to use that format.
YYYYMMDD-HHMM is most widely understood without spaces IMO